src/app/ssmt/operations/operations.service.ts

```diff
@@ -53,4 +55,44 @@ export class OperationsService {
       generalSteamOperations: generalSteamOperations
     }
   }
+
+  checkOperationsWarnings(operationsForm: FormGroup, ssmt: SSMT, settings: Settings): BoilerWarnings {
+    return {
+      makeUpWaterTemperature: this.checkMakeupWaterTemperatureWarnings(operationsForm, ssmt, settings)
+    };
+  }
+
+  checkMakeupWaterTemperatureWarnings(formGroup: FormGroup, ssmt: SSMT, settings: Settings) {
+    let warning = null;
+    if (ssmt.boilerInput.preheatMakeupWater == true) {
```

Comment:
boilerInput doesn't exist on ssmt until visiting the boiler page so errors blow up new assessments on first visit to operations page

should just need to check it exists